### DISCUSSION TITLE
Trouble getting to minimum UI Test Coverage

### DISCUSSION THREAD

**Discussion by BKrajancich** [CS342/.github#26] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

I'm having trouble getting my code coverage up to min. 90% because XCode takes the PreviewProvider into account.

What should I do? Remove all the SwiftUI previews? Or is there a way I can exclude some lines with 'PreviewProvider' keywords etc...

### Reproduction

Deploy App

### Expected behavior

Want the UI tests to pass

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by BKrajancich** [comment, marked as answer]
I ended up fixing it with this:
https://www.christopherthiebaut.com/posts/exclude_swiftui_previews_from_code_coverage/

**Reply by PSchmiedmayer** [reply]
A very good solution @BKrajancich, very cool! I will also see that I will use that in the template application.
